public/content/en/basics.md

```diff
@@ -745,30 +745,45 @@ void main() {
     writeln("My dstring: ", dstr);
 }
 
-# All the classic for's
+# For loops
```

Comment:
The next section is called just foreach so "Loops" would be fine IMO. C-style loops is maybe awkward for someone coming from a non-C backward.